crates/stc_ts_errors/src/lib.rs

```diff
@@ -1788,6 +1788,8 @@ impl ErrorKind {
             ErrorKind::TypeInvalidForUpdateArg { .. } => 2356,
             ErrorKind::ExprInvalidForUpdateArg { .. } => 2357,
 
+            ErrorKind::PrivatePropertyIsDifferent { .. } => 2415,
```

Comment:
i thought about creating a new ErrorKind here but it made sense to reuse it. it previously was used for emit 2420 for incorrectly implementing interfaces. so this might be wrong